finished/wrangler.jsonc

```diff
@@ -31,26 +18,20 @@
       }
     ]
   },
-
-  // Environment variables
   "vars": {
     "APP_NAME": "applywize"
   },
-
-  // Migrations configuration
   "migrations": [
     {
       "tag": "v1",
-      "new_classes": ["SessionDurableObject"]
+      "new_sqlite_classes": ["SessionDurableObject"]
```

Comment:
This was the only other change in this file other than  the db change below https://github.com/ahaywood/applywize/pull/1/files#r2055560761